secretstores/gcp/secretmanager/secretmanager_test.go

```diff
@@ -18,6 +18,7 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"golang.org/x/net/context"
```

Comment:
Use `context` from the standard library instead.

Reply:
done.